BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Views/SkontoWithDiscountPriceView.swift

```diff
@@ -25,12 +25,17 @@ class SkontoWithDiscountPriceView: UIView {
         super.init(frame: .zero)
         setupView()
         bindViewModel()
+        setupKeyboardObservers()
     }
 
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
 
+    deinit {
+        NotificationCenter.default.removeObserver(self)
```

Comment:
[suggestion] Maybe is better to remove that specific observer that we added like:
`  NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillHideNotification, object: nil)` what do you say?

Reply:
I think on deinit it's better always to remove self from NotificationCenter. We don't have object anymore, so it will be handled with 1 line of code for all observables. Removing some specific - when we already handled some logic and we don't need updates from NotificationCenter but object is alive, not deinited and can receive some other notifications, but from another observables